src/components/login-form.js

```diff
@@ -79,37 +75,16 @@ export class LoginForm extends React.Component {
               {this.state.message}
             </div>
           ) : (
-            <div className="alert alert-danger" role="alert">
-              {this.state.message}
-            </div>
-          ))}
+              <div className="alert alert-danger" role="alert">
+                {this.state.message}
+              </div>
+            ))}
         <form>
-          <div className="form-group">
-            <label htmlFor="exampleInputEmail1">Email address</label>
-            <input
-              type="email"
-              className="form-control"
-              id="exampleInputEmail1"
-              aria-describedby="emailHelp"
-              placeholder="Enter email"
-              onChange={this.onChangeEmail}
-              value={this.state.email}
-            />
-            <small id="emailHelp" className="form-text text-muted">
-              We'll never share your email with anyone else.
-            </small>
-          </div>
-          <div className="form-group">
-            <label htmlFor="exampleInputPassword1">Password</label>
-            <input
-              type="password"
-              className="form-control"
-              id="exampleInputPassword1"
-              placeholder="Password"
-              onChange={this.onChangePassword}
-              value={this.state.password}
-            />
-          </div>
+          <Input type="email" id="email" value={this.state.email} placeholder="Enter email" label="Email address"
```

Comment:
Having all props of a component in the same line is not legible, plus it makes diffs hard to read / parse (errors can be introduced easily)

Please have each prop in a separate line

Reply:
Fixed, each prop in a single line.